src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,36 @@
+.calendar {
+  width: $calendar-width;
+  display: flex;
+  gap: $cell-gap;
+  flex-wrap: wrap;
+  &__cell {
+    transition: transform 0.5s;
+    cursor: pointer;
+    @include centeredSquare;
+    @for $i from 1 through 31 {
+      &:nth-child(#{$i})::before {
+        content: "#{$i}";
+      }
+    }
+    border: $cell-border;
+    background-color: $cell-color;
+    &:hover {
+      background-color: pink;
+      transform: translateY(-20px);
+    }
+  }
+}
```

Comment:
Don't forger about the empty line between classes
also, create variable for $transitionDuration
```suggestion
  flex-wrap: wrap;
  
  &__cell {
    transition: transform $transitionDuration;
    cursor: pointer;
    
    @include centeredSquare;
    
    @for $i from 1 through 31 {
      &:nth-child(#{$i})::before {
        content: "#{$i}";
      }
    }
    
    border: $cell-border;
    background-color: $cell-color;
    
    &:hover {
      background-color: pink;
      transform: translateY(-20px);
    }
  }
}
```